DNN Platform/DotNetNuke.Abstractions/Portals/IPortalStyles.cs

```diff
@@ -0,0 +1,227 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Portals
+{
+    /// <summary>
+    /// Provides css custom properties to customize the Dnn UI in the platform, themes and modules for an overall consistent look.
+    /// </summary>
+    public interface IPortalStyles
+    {
+        /// <summary>
+        /// Gets or sets the main shade of the primary color.
+        /// </summary>
+        IStyleColor ColorPrimary { get; set; }
+
+        /// <summary>
+        /// Gets or sets the light shade of the primary color.
+        /// </summary>
+        public IStyleColor ColorPrimaryLight { get; set; }

```

Comment:
These shouldn't have `public` since it's an interface, right?

Reply:
Good catch, fixed in https://github.com/dnnsoftware/Dnn.Platform/pull/5851/commits/396c252c0f7b05533356b27f454094b7abe30d01